tiledb/sm/array_schema/array_schema.h

```diff
@@ -297,6 +299,28 @@ class ArraySchema {
   /** Returns the array schema version. */
   uint32_t version() const;
 
+  /** Set a timestamp range for the array schema */
+  Status set_timestamp_range(
+      const std::pair<uint64_t, uint64_t>& timestamp_range);
+
+  /** Returns the timestamp range */
+  const std::pair<uint64_t, uint64_t>& timestamp_range() const;
```

Comment:
Don't return a const-ref here, just let this make a copy with a normal return value. Same elsewhere (e.g. uri() and name()), we want to avoid returning internal data unless we absolutely need to.

Reply:
done. changed to return deep copy of them